qctrlopencontrols/__init__.py

```diff
@@ -14,14 +14,21 @@
 
 """
 =================
-qcrtlopencontrols
+qctrlopencontrols
 =================
 """
 
-from .dynamic_decoupling_sequences import (DynamicDecouplingSequence,
-                                           new_predefined_dds,
-                                           convert_dds_to_driven_control)
-from .driven_controls import DrivenControl, new_predefined_driven_control
-from .qiskit import convert_dds_to_quantum_circuit
-from .cirq import (convert_dds_to_cirq_circuit,
-                   convert_dds_to_cirq_schedule)
+from . import dynamic_decoupling_sequences
+from .dynamic_decoupling_sequences import *
```

Comment:
I'm not sure about this -- it means the user has to look into the sub-packages to see the API. Although I suppose they have to do that anyway, to look at the method documentation...

How verbose will it be if we list out all the imports explicitly?

Reply:
This is related to the other 2 comments [above and below this one]; if we decide __all__ goes only in the top level, this should take care of this problem; Otherwise, I was following numpy case e.g. https://github.com/numpy/numpy/blob/master/numpy/__init__.py; again happy to go with either decision.

Reply:
OK, happy to be consistent with numpy. If we find it awkward going forwards we can always change it.